relevanceai/operations_new/cluster/base.py

```diff
@@ -34,6 +35,17 @@ def __init__(
         for k, v in kwargs.items():
             setattr(self, k, v)
 
+    def _check_vector_fields(self):
+        # check the vector fields
+        for vector_field in self.vector_fields:
+            if not vector_field.endswith("_vector_"):
+                raise ValueError("Vector field is bad!")
```

Comment:
This error is uninformative. should be more specific:
f'{vector_field} is not a valid vector field as it does not end with "_vector" ', 

This also needs to check for length. if vector fields is length of 0 and if the vector field is a string or not.

Reply:
This should also sit in the operations base not just cluster base